Author: Reiner Herrmann <reiner@reiner-h.de>
Description: Don't regenerate version.cmake
 use version.cmake from the release tarball, instead of letting it regenerate
 when a git directory has been detected.
Forwarded: not-needed

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -434,7 +434,6 @@
     set(VERSION_STRING_GIT "${MAJOR_VERSION_GIT}.${MINOR_VERSION_GIT}.${PATCH_VERSION_GIT}")
   endif()
 
-  configure_file("${CMAKE_SOURCE_DIR}/version.cmake.in" "${CMAKE_SOURCE_DIR}/version.cmake")
 endif()
 if(NOT EXISTS "${CMAKE_SOURCE_DIR}/version.cmake")
   message( SEND_ERROR "Could not find GIT or valid version.cmake. Version information will be invalid." )
